docker-compose.yml

```diff
@@ -36,25 +30,23 @@ services:
   load_db:
     build: ./load_db
     volumes:
-    - ./data:/data
+    - input_data:/data/input
+    - imposm_generated_data:/data/generated
+    - expired_tiles:/data/expired_tiles
```

Comment:
I am not sure about the name "expired_tiles".  
The update script will probably need to store some other files that expired tiles to save the current state of the update.
What about "update_data" ? :thinking: